src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -47,6 +52,28 @@ public async Task<IActionResult> GetQuestionBySlug(string sectionSlug,
         return await router.ValidateRoute(sectionSlug, questionSlug, this, cancellationToken);
     }
 
+    [LogInvalidModelState]
+    [HttpGet("question/preview/{questionId}")]
+    public async Task<IActionResult> GetQuestionPreviewById(string questionId,
+                                                            [FromServices] ContentfulOptions contentfulOptions,
+                                                            CancellationToken cancellationToken = default)
+    {
+        if (!contentfulOptions.UsePreview)
+            return new RedirectResult("/self-assessment");
+
+        var question = await _getEntityFromContentfulQuery.GetEntityById<Question>(questionId, cancellationToken) ??
+                       throw new KeyNotFoundException($"Could not find question with Id {questionId}");
+
+        var section = await _getSectionQuery.GetSectionForQuestion(questionId, cancellationToken) ??
```

Comment:
If we're just rendering the question page, what do we need all the section details for?

Reply:
We don't, the section stuff is purely for the form, because submit answer needs it, and the preview reuses the question page - I was considering making a `QuestionPreview.cshtml` page that removes the form part, so that we also prevent people from submitting answers from the preview page, but it seems overkill when it doesn't really matter if a CD presses submit because they can and gets an error message 

Butttt, what I could do, is just making section an optional param for the view model and pass it nothing in this case
and then delete all this, so I like this plan, I'll do it

Reply:
I just read your other comment, I reckon, I can skip the form if section is not passed in and thereby make submitting impossible